main.ledger

```diff
@@ -2054,6 +2100,29 @@
     Income:Investments:Bank Interest
     ; Receipt: e8c545b4f47c9c0b249c9b77e1278172.png
 
+2016/05/31 Google
+    Expenses:Services:Software                          $28.00
+    Assets:Bank:Checking
+    ; Receipt: 85d83bb6a67cac4cb0372de340701753.pdf
+    ; TODO is this reimbursement
+
+2016/05/31 Facebook Adverts
+    Expenses:Services:Software                           $5.49
+    Liabilities:Reimbursement:Max Wofford
+    ; Receipt: 42dc4be59676aea7604e6d8556bd5083.pdf
+
+2016/06/00 Uber
+    Expenses:Transport:Rental Car                        $4.75
+    Liabilities:Reimbursement:Max Wofford
+    ; Receipt: 7cf312fb13eb916c18bfcad320559e00.pdf
+    ; TODO clarify date
```

Comment:
This was June 6th. I've added a copy of the receipt with the date in my reimbursements folder that you can replace this with.